.github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md

```diff
@@ -0,0 +1,54 @@
+---
+name:  Dependencies Issue
+about: Use this template for all Dependencies Issues, even those automatically generated
+labels: dependencies
+---
+
+# Description
+- Information about the dependency and its role in the project
+- Current version - what are we running
+- Target Version - what are we changing it to
+- DependaBot PR link if available
+
+## Features and fixes available
+- List of important bug fixes, features or security patches
+- List of the important changes in this version that could affect the workflow
+
+## Additional information
+- Additional information about the version update.
+- Guideline that can be followed for the process
+
+## QA Needs
+< Indicate whether this item needs QA testing. For those that do, indicate any special setup or tool set the QA team might need. >
+
+## Acceptance Criteria
```

Comment:
I feel like this should be the same across the board. Dependency is updated, and UX is not altered

Reply:
hmm, lost another set of comments. Probably too senile to hit "Comment" which really should be "Save" since I already commented by writing in the box... Anywho, testing these generally relies on a sanity test of whatever area of the system QA deems necessary to test but usually with a lot of input from Dev as to what has likely been broken, er changed. I think there will be lots of cases where nothing special is needed but for those that do, it's worth it to have this section. Can I leave it?